src/components/tag/tag.js

```diff
@@ -120,7 +120,7 @@ const TagBody = props => (
         `,
       props.styles.body,
     ]}
-    onClick={props.isDisabled ? undefined : props.onClick}
```

Comment:
Wrong one 😆 

I think we still need this one because it's not an `AccessibleButton`